src/components/index.tsx

```diff
@@ -7,10 +9,13 @@ import AppointmentTableContainer from './appointmentsTable';
 import { TherapistCardProps, TherapistCardType } from './TherapistCard/types';
 import GridCard from './TherapistCard';
 import TherapistList from './TherapistsList';
+import BioEditor from './therapistProfile/changeBio';
```

Comment:
can we import all components from therapistProfile from index file?
```suggestion
import { BioEditor } from './therapistProfile/';
```